### ISSUE TITLE
Missing Text in VS2017 source code

### ISSUE THREAD

**Issue by YetiAnt** [nvaccess/nvda#8975]
In C# source code You can Open or Close code blocks by pressing 

   STRG+M,M

This works fine in NVDA with VS2017 with blocks like namespaces, classes and Methods.

This feature is also available for regions. But it does not work properly with NVDA.

You may reproduce the issue as follows:
1. Open any C# project in VS2017. You can use the attached NVDAVS2017RegionColapseProblem.
2. Define a region. For example: 
      // Begin Issue
      #region Issue
      Console.Write("This Program does not do anything! Just press Enter to close");
      Console.ReadLine();
      #endregion Issue
      // End Issue
3. Now move carrot to the line: “#region Issue”
4. Press STRG+M,M to collapse the region. 
After this it should look like this 
      // Begin Issue
      Issue
      // End Issue
But in Fact it looks like 
      // Begin Issue
      
      // End Issue

Remarks:
* The lines beginning with “//” have no meaning for the code. They are only for explanation.
* This issue in fact may be no one of NVDA, but one of VS2017. If so, it should be forwarded to Microsoft.

Many thanks already now for reading 

Yeti

Further Information:
- NVDA 2018.3.2 and earrlyer - installed 
- Windows 10 X64 Version 10.0 Build: 17134 PlatformID: 2 SP
- Microsoft Visual Studio Community 2017 
-- Version 15.5.7 AND EARLYER
-- VisualStudio.15.Release/15.5.7+27130.2036
-- Microsoft .NET Framework Version 4.7.03056 and earlyer 
-- Visual C# 2017   00369-60000-00001-AA835
- The issue is constant since I installed VS2017 in spring 2018

**Comment by Adriani90** [comment]
VS 2017 mainstream support ended in 2022, there are only security updates provided by Microsoft. I don't expect any further accessibility updates to VS 2017, so I am closing as abandoned. If you have this issue in a newer version of VS Studio, Please open a new issue or comment on this and we can reopen.
